bot/exts/info/doc/_parsing.py

```diff
@@ -256,4 +256,9 @@ def get_symbol_markdown(soup: BeautifulSoup, symbol_data: DocItem) -> Optional[s
         signature = get_signatures(symbol_heading)
         description = get_dd_description(symbol_heading)
 
+    for description_element in description:
+        if isinstance(description_element, Tag):
+            for tag in description_element.find_all("a", class_="headerlink"):
```

Comment:
Can't you re-use `_filter_signature_links` in this case?

Reply:
I suppose it could be used, but I'm not convinced if it'd be worth it. 
Using the filter would only change the find_all call which is already clear enough, while also introducing a bit more work to the filtering and adding an another import for it

Reply:
Fair enough.